charts/posthog/templates/_posthog.tpl

```diff
@@ -8,6 +8,14 @@
   {{- end -}}
 {{- end }}
 
+{{- define "posthog.clusterName" }}
```

Comment:
I don't see much value of having this if we anyway specify a default in the values